web-services/deploy/docs/docs/query_help.html

```diff
@@ -209,5 +209,17 @@ <h3>LUCENE Query Syntax</h3>
 		</ul>
 		<li>A special date format of 'e' can be supplied to mean milliseconds since epoch.</li>
 	</ol>
+	<h3>Unique Functions</h3>
+	<p class="left">In addition, there are functions that are available to visit the query tree, and perform operations on the nodes and their
+		children. The extracted parameters will be added to the provided data. Any options function nodes will be subsequently deleted and remaining AND/OR nodes will be optimized to account for the potential removal of their children.
+		Potential function options that the user may use are listed below.</p>
+	<ul>
+		<li>f:options(): Expects a comma-delimited list of key/value pairs, e.g. f:options('hit.list','true','limit.fields','FOO_1_BAR=3)</li>
```

Comment:
This part I think is more appropriate. I like the examples. You might want to add a follow on paragraph with an example or something for how "unique_by_x" impacts the results.

For instance, let's say we have

```
event1, 01/01/2022, icecream=vanilla
event2, 01/01/2022, icecream=vanilla
event3, 01/01/2022, icrecream=chocolate

event4, 01/02/2022, icecream=vanilla
```
Unique_by_day, I'm assuming could either yield (event1, event3, event4) OR (event1, event2, event4). You could also add minutes into that example.

Reply:
Thanks. I'll look to build off of this and put in some more concrete examples.